static/css/around-map.css

```diff
@@ -116,6 +116,8 @@ footer {
   background: rgba(0, 0, 0, 0.6);
   z-index: 2;
   overflow: scroll;
+  z-index: 10;
```

Comment:
When something so specific comes up, just give the code a comment explaining why.  Like `/* Above header */`